fdroid/repo/index-v2.json

```diff
@@ -4,7 +4,7 @@
       "en-US": "Bitwarden F-Droid"
     },
     "description": {
-      "en-US": "This is a repository of Bitwarden apps to be used with F-Droid. Applications in this repository are official binaries built by Bitwarden."
+      "en-US": "This is a repository of Bitwarden apps to be used with F-Droid. Applications\nin this repository are official binaries built by Bitwarden.\n"
```

Comment:
Are these `\n` expected?

Reply:
It is, because we have them in `base_fdroid_config.yml`. They're not necessary though. We can change the config and they'll be removed on the next scheduled run.